### DISCUSSION TITLE
Missing method on lucid query builder

### DISCUSSION THREAD

**Discussion by eliotovele** [adonisjs/core#2118] [General]
Hi, how can return array of values for the selected column?, in version 4 I used the **pluck** method, but it seems that the **pluck**  method disappeared in version 5, any suggestions?

**Comment by RomainLanz** [comment]
Hey @eliotovele! 👋 

AdonisJS 5 returns an array of models.
You can use the `Array#map` function to pluck the columns you want.

**Comment by eliotovele** [comment]
@RomainLanz  thank you, it worked.
